### PR TITLE
ci: fix deprecation of github actions

### PR DESCRIPTION
**Pull request by JoanFM**
Make sure we adapt to future deprecations from Github

### CODE REVIEW

File: .github/workflows/cd.yml [hunk 1]
```diff
@@ -102,7 +102,8 @@ jobs:
       - id: set-matrix
         run: |
           sudo apt-get install jq
-          echo "::set-output name=matrix::$(bash scripts/get-all-test-paths.sh)"
+          value = $(bash scripts/get-all-test-paths.sh)"
```
Comment (JoanFM) [inline .github/workflows/cd.yml:105]: @deepankarm u know how this is done in bash?

File: jina_cli/autocomplete.py [hunk 2]
```diff
@@ -212,8 +212,7 @@
             '--force',
         ],
         'hub status': ['--help', '--id', '--verbose', '--replay'],
-        'hub list': ['--help'],
-        'hub': ['--help', 'new', 'push', 'pull', 'status', 'list'],
+        'hub': ['--help', 'new', 'push', 'pull', 'status'],
```
Comment (JohannesMessner) [inline jina_cli/autocomplete.py:215]: Is this related to these other changes?

Comment (JoanFM) [reply jina_cli/autocomplete.py:215]: updated branch should solve

**Review by JohannesMessner — APPROVED** [review]
